desafio-03/JamesStewart-314/python/solution.py

```diff
@@ -0,0 +1,63 @@
+"""
+|-------------------------------------|
+        Author: @JamesStewart-314
+  Minimum python version required: 3.10
+|-------------------------------------|
+"""
+
+from typing import Callable, TypeAlias, Final
+
+NumberConstraint: TypeAlias = tuple[int | None, int | None] | None
+
+MAXINTSIZE: Final[int] = (1 << 64) - 1
+
+number_is_palindromic: Callable[[int], bool] = lambda x: (strNumber := str(x)) == strNumber[::-1]
+
+def validade_number_constraints(number: int,
+                                constraints: NumberConstraint) -> None:
+    if constraints is None:
+        return
+    if constraints[0] is not None and number < constraints[0]:
+        raise ValueError(f"Last input number must be greater"\
+                         f" or equal to '{constraints[0]}'.")
+    if constraints[1] is not None and number > constraints[1]:
+        raise ValueError(f"Last input number must be less"\
+                         f" or equal to '{constraints[1]}'.")
+    return
+
+
+def get_valid_number(message: str | None = None,
+                     constraints: NumberConstraint = None) -> int:
+    while True:
+        try:
+            userInput: int = int(input(message if message else ""))
```

Comment:
Algumas variáveis como `strNumber`, `userInput`, `palindromicNumbersFound`, `firstNumber`, `secondNumber`, não estão conforme o estilo de nomenclatura `snake_case`

[Function and Variable Names](https://peps.python.org/pep-0008/#function-and-variable-names)